internal/cli/organizations.go

```diff
@@ -77,6 +77,13 @@ var (
 		Help:       "Role Identifier.",
 		IsRequired: true,
 	}
+
+	organizationNumber = Flag{
+		Name:      "Number",
+		LongForm:  "number",
+		ShortForm: "n",
+		Help:      "Number of organizations, that match the search criteria, to retrieve. Maximum result number is 1000.",
```

Comment:
This help text appears for all subcommands of `auth0 orgs`, not sure if it's best to also define separate flags for `auth0 orgs roles` and `auth0 orgs members`

Reply:
Don't forget to add a comment indicating why the `organizationNumber` var does not have a help text, for future reference (otherwise on a skim it might seem like it's just missing).

Reply:
Done in c405a2e7aeb28fe2ac61591b20db278be64a27f6